packages/examples/src/walk-on-spheres/Laplace.sub

```diff
@@ -0,0 +1,20 @@
+-- variation: GroundedMule26071
+
+-- Describe a short walk used to estimate the solution to
+-- a basic Laplace equation Δu = 0, which involves both
+-- boundary samples x and source samples y.
+
+Domain U
+Step x0, x1, x2, x3, x4
+
+x1 := sampleBoundary( x0 )
+x2 := sampleBoundary( x1 )
+x3 := sampleBoundary( x2 )
+x4 := sampleBoundary( x3 )
```

Comment:
This is just a matter of aesthetics, but could we do this instead? (same for the other `.sub` files in this directory)
```suggestion
Step x0

Step x1 := sampleBoundary( x0 )
Step x2 := sampleBoundary( x1 )
Step x3 := sampleBoundary( x2 )
Step x4 := sampleBoundary( x3 )
```